core/src/crds_gossip_pull.rs

```diff
@@ -468,11 +467,10 @@ impl CrdsGossipPull {
         bloom_size: usize,
     ) -> Vec<CrdsFilter> {
         const PAR_MIN_LENGTH: usize = 512;
-        let num = cmp::max(
-            CRDS_GOSSIP_DEFAULT_BLOOM_ITEMS,
-            crds.len() + self.purged_values.len() + self.failed_inserts.len(),
-        );
-        let filters = CrdsFilterSet::new(num, bloom_size);
+        const MIN_NUM_BLOOM_ITEMS: usize = 65_536;
```

Comment:
Will this be a moving target as the number of validators on the network increases?

Reply:
We will see, but probably with https://github.com/solana-labs/solana/pull/17261 we should be fine